pydruid/client.py

```diff
@@ -420,17 +581,68 @@ def _post(self, query):
             query.query_id = res.info().get('x-druid-query-id')
             data = content.decode("utf-8")
             res.close()
-        except urllib.error.HTTPError:
-            _, e, _ = sys.exc_info()
-            err = None
+        except urllib.error.HTTPError as e:
+            err = e.read()
             if e.code == 500:
                 # has Druid returned an error?
                 try:
-                    err = json.loads(e.read().decode("utf-8"))
+                    err = json.loads(err)
```

Comment:
not sure why we don't decode? anyway I see that it's already been that way so fine

Reply:
hmm not sure either
I think it was part of the merge
will have a look